src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -569,3 +571,32 @@ def snapshot_schedule_add(cls, image_spec: str, interval: str):
     @classmethod
     def snapshot_schedule_remove(cls, image_spec: str):
         _rbd_support_remote('mirror_snapshot_schedule_remove', image_spec)
+
+
+class RbdImageMetadataService(object):
+    def __init__(self, image):
+        self._image = image
+
+    def list(self):
+        result = self._image.metadata_list()
+        # filter out configuration metadata
+        return {v[0]: v[1] for v in result if not v[0].startswith('conf_')}
```

Comment:
Why do we want to filter out this metadata?

Reply:
Image metadata is also used to store image configuration (metadata keys with 'conf_' prefix), and we already return this in image info as 'configuration'. So I think it will be redundant and look rather confusing if we return the same information twice.